tools/go.mod

```diff
@@ -0,0 +1,173 @@
+module github.com/code-ready/crc/tools
+
+go 1.18
```

Comment:
maybe we should keep the same go version as the one used for crc here? `1.17`

Reply:
Yes, definitely, thanks for catching this! I've repushed with this changed, and will look at your m1 issue next.